hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/InputPathHandler.java

```diff
@@ -50,20 +59,22 @@
   private final Map<HoodieTableMetaClient, List<Path>> groupedIncrementalPaths;
   private final List<Path> snapshotPaths;
   private final List<Path> nonHoodieInputPaths;
+  private final JobConf jobConf;
 
-  public InputPathHandler(Configuration conf, Path[] inputPaths, List<String> incrementalTables) throws IOException {
+  public InputPathHandler(Configuration conf, Path[] inputPaths, List<String> incrementalTables, JobConf jobConf) throws IOException {
```

Comment:
If we remove `HoodieTableGloballyConsistentMetaClient.java` is there a way to avoid passing in `JobConf` ?